### ISSUE TITLE
Modal prompt missing ARIA stuff

### ISSUE THREAD

**Issue by chriscorwin** [salesforce/design-system-react#382]
Discovered in #352 

Need to add `aria-labelledby="prompt-heading-id" aria-describedby="prompt-message-wrapper" role="alertdialog"` to the `aria-hidden="false"` wrapper around the modal container, and the supporting HTML elements within it:
## Expected output

``` html
<div>
  <div aria-hidden="false" aria-labelledby="prompt-heading-id" aria-describedby="prompt-message-wrapper" role="alertdialog" class="slds-modal slds-modal--prompt slds-fade-in-open">
    <div class="slds-modal__container slds-modal--prompt" role="document" id="prompt-message-wrapper" tabindex="0">
      <div class="slds-modal__header slds-theme--error slds-theme--alert-texture">
        <h2 class="slds-text-heading--medium" id="prompt-heading-id">Service Unavailable</h2>
      </div>
      <div class="slds-modal__content slds-p-around--medium">
        <div>
          <p>Sit nulla est ex deserunt exercitation anim occaecat. Nostrud ullamco deserunt aute id consequat veniam incididunt duis in sint irure nisi.</p>
        </div>
      </div>
      <div class="slds-modal__footer slds-theme--default">
        <button class="slds-button slds-button--neutral">Okay</button>
      </div>
    </div>
  </div>
  <div class="slds-backdrop slds-backdrop--open"></div>
</div>
```
